PIVX-Labs/MyPIVXWallet
# Wallet Balance GUI

## Abstract

Rewrite the wallet balance GUI in vue

## Testing

- Check that the balance updated correctly and displays the correct value in the selected currency
- Check that the GUI updates when a new currency is selected
- Check that the three dots menu items work and display correctly (i.e. Hardware wallets shouldn't have an export key item, legacy wallets don't have a refresh address item)
